Alexandra/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Tea shop</title>
+</head>
+
+<body>
+    <div class="container">
+        <nav id="topBar">

```

Comment:
each id should match this naming convention: `something-here`, so don't use `somethingHere`. That is mostly used in javascript, not html or css